sale_automatic_workflow_payment_mode/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Sodexis <dev@sodexis.com>
```

Comment:
You can add Tecnativa here with your name :)

Also, you can re-generate the README if you change it.